### ISSUE TITLE
use case: track all idigbio records/images related to bee family Andrenidae

### ISSUE THREAD

**Issue by jhpoelen** [bio-guoda/preston#75]
as suggested by @seltmann in relation to #71 .

Suggested to expose properties like:

dwc:dateIdentified | dwc:identificationQualifier | dwc:identifiedBy | dwc:lifeStage | dwc:nameAccordingTo | ac:otherScientificName | dwc:preparations | dwc:scientificName | dwc:scientificNameID | dwc:sex | ac:subjectOrientation | ac:subjectPart | ac:taxonCount | ac:taxonCoverage | dwc:vernacularName

workflow:

1. enter search criteria
2. clone idigbio using Preston and jekyll 
3. run local site using cloned content
4. periodically, register all images and update the content from idigbio

**Comment by jhpoelen** [comment]
related Preston command:

```
preston track https://search.idigbio.org/v2/search/records/?rq=%7B%22family%22%3A%22Andrenidae%22%2c%22hasImage%22%3A%22true%22%7D
```

note that 
```
>> decodeURIComponent('rq=%7B%22family%22%3A%22Andrenidae%22%2c%22hasImage%22%3A%22true%22%7D')
< "rq={\"family\":\"Andrenidae\",\"hasImage\":\"true\"}"
```

**Comment by seltmann** [comment]
Expose https://www.tdwg.org/standards/ac/ which contains information about the image.

**Comment by jhpoelen** [comment]
Hey @seltmann - I created a jekyll site based on automatically tracked content from iDigBio, added the content to https://github.com/jhpoelen/bees and hosted via Github pages at https://jhpoelen.nl/bees . With this, you can access iDigBio records, metadata and linked images offline, or host it yourself. You should be able to build your own site with the instructions on https://jhpoelen.nl/bees . Can you try? 

Note that the site itself is self-replicating: you can "clone" the hosted data using preston, then generate another jekyll site etc. Also, you can update the data, by re-running the ```preston track ...``` command to pull in more records and update existing records.

In other words, the site at https://jhpoelen.nl/bees is generated from the data itself. And the data is location agnostic. We can use the same approach to publish / archive / distribute Parasite Tracker content indexed via iDigBio . . . as a way to keep track of what is available. 

Curious to hear your thoughts! 

ps - the web ui is auto-generated and using the jekyll / liquid templates. The generated pages are generated from structured data and the look 'n feel can be adopted to your own project.

**Comment by jhpoelen** [comment]
Prototype was reviewed in person with @seltmann @mjy @qgroom @mielliott and others.

This prototype resulted in another use case: build image corpus for Herbarium Martii label detection algorithm see #80 and https://github.com/bio-guoda/preston/issues/80 .
